internal/util/plugin_schema_helper.go

```diff
@@ -7,9 +7,6 @@ import (
 	"github.com/kong/go-kong/kong"
 )
 
-// FIXME
```

Comment:
Why are we removing this warning?

Reply:
This I believe was relevant for the swap of `GetSchema()` to `GetFullSchema()` (mark that `GetSchema()` is deprecated). Hence there's nothing else really to fix here from my PoV. 

BTW all this code is really doing here is memoizing the plugin schemas 🤷